src/__tests__/FileUpload.test.tsx

```
@@ -0,0 +1,850 @@
+      const file = new File(['content'], 'dropped.txt', { type: 'text/plain' });
+      
+      const dropEvent = new Event('drop', { bubbles: true });
+      Object.defineProperty(dropEvent, 'dataTransfer', {
+        value: {
+          files: [file],
+          items: [],
+          types: ['Files'],
+        },
+      });
+      
+      fireEvent(dropZone!, dropEvent);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)